packages/json-serializer/test.js

```diff
@@ -1,11 +1,50 @@
,{"pos":{"_x":1,"_y":1,"_z":1},"tag":15},{"pos":{"_x":1,"_y":-1,"_z":1},"tag":12}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":1,"_y":0,"_z":0},"w":1,"tag":3}},{"vertices":[{"pos":{"_x":-1,"_y":-1,"_z":-1},"tag":9},{"pos":{"_x":1,"_y":-1,"_z":-1},"tag":13},{"pos":{"_x":1,"_y":-1,"_z":1},"tag":12},{"pos":{"_x":-1,"_y":-1,"_z":1},"tag":10}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":-1,"_z":0},"w":1,"tag":4}},{"vertices":[{"pos":{"_x":-1,"_y":1,"_z":-1},"tag":8},{"pos":{"_x":-1,"_y":1,"_z":1},"tag":11},{"pos":{"_x":1,"_y":1,"_z":1},"tag":15},{"pos":{"_x":1,"_y":1,"_z":-1},"tag":14}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":1,"_z":0},"w":1,"tag":5}},{"vertices":[{"pos":{"_x":-1,"_y":-1,"_z":-1},"tag":9},{"pos":{"_x":-1,"_y":1,"_z":-1},"tag":8},{"pos":{"_x":1,"_y":1,"_z":-1},"tag":14},{"pos":{"_x":1,"_y":-1,"_z":-1},"tag":13}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":0,"_z":-1},"w":1,"tag":6}},{"vertices":[{"pos":{"_x":-1,"_y":-1,"_z":1},"tag":10},{"pos":{"_x":1,"_y":-1,"_z":1},"tag":12},{"pos":{"_x":1,"_y":1,"_z":1},"tag":15},{"pos":{"_x":-1,"_y":1,"_z":1},"tag":11}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":0,"_z":1},"w":1,"tag":7}}],"isCanonicalized": true,"isRetesselated": true}','{ "type": "csg","polygons": [{"vertices":[{"pos":{"_x":-5,"_y":-5,"_z":-5},"tag":23},{"pos":{"_x":-5,"_y":-5,"_z":5},"tag":24},{"pos":{"_x":-5,"_y":5,"_z":5},"tag":25},{"pos":{"_x":-5,"_y":5,"_z":-5},"tag":22}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":-1,"_y":0,"_z":0},"w":5,"tag":16}},{"vertices":[{"pos":{"_x":5,"_y":-5,"_z":-5},"tag":27},{"pos":{"_x":5,"_y":5,"_z":-5},"tag":28},{"pos":{"_x":5,"_y":5,"_z":5},"tag":29},{"pos":{"_x":5,"_y":-5,"_z":5},"tag":26}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":1,"_y":0,"_z":0},"w":5,"tag":17}},{"vertices":[{"pos":{"_x":-5,"_y":-5,"_z":-5},"tag":23},{"pos":{"_x":5,"_y":-5,"_z":-5},"tag":27},{"pos":{"_x":5,"_y":-5,"_z":5},"tag":26},{"pos":{"_x":-5,"_y":-5,"_z":5},"tag":24}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":-1,"_z":0},"w":5,"tag":18}},{"vertices":[{"pos":{"_x":-5,"_y":5,"_z":-5},"tag":22},{"pos":{"_x":-5,"_y":5,"_z":5},"tag":25},{"pos":{"_x":5,"_y":5,"_z":5},"tag":29},{"pos":{"_x":5,"_y":5,"_z":-5},"tag":28}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":1,"_z":0},"w":5,"tag":19}},{"vertices":[{"pos":{"_x":-5,"_y":-5,"_z":-5},"tag":23},{"pos":{"_x":-5,"_y":5,"_z":-5},"tag":22},{"pos":{"_x":5,"_y":5,"_z":-5},"tag":28},{"pos":{"_x":5,"_y":-5,"_z":-5},"tag":27}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":0,"_z":-1},"w":5,"tag":20}},{"vertices":[{"pos":{"_x":-5,"_y":-5,"_z":5},"tag":24},{"pos":{"_x":5,"_y":-5,"_z":5},"tag":26},{"pos":{"_x":5,"_y":5,"_z":5},"tag":29},{"pos":{"_x":-5,"_y":5,"_z":5},"tag":25}],"shared":{"color":null,"tag":2},"plane":{"normal":{"_x":0,"_y":0,"_z":1},"w":5,"tag":21}}],"isCanonicalized": true,"isRetesselated": true}' ]
```

Comment:
our json structures are just horrible, hurray for v2 !

Reply:
It was kind of a hack, but works well.